db/migrate/20220811013750_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :author, null: false, foreign_key: { to_table: 'users' }
+      t.references :post, null:false, foreign_key: { to_table: 'posts' }
+      t.string :text
```

Comment:
- [ ] Here also it would be better to use `t.text` instead of `t.string` for `:text`.